### ISSUE TITLE
Class Sub-Member access

### ISSUE THREAD

**Issue by JoePelz** [riolet/ritchie#21]
The following code should work, but doesn't:
```
Point :: BaseType
	x = Integer

Range :: BaseType
  start = Point 

  :printout
    print start.x
```
The issue is that currently `start.x` gets translated as `start->x`
It should be translated as `self->start->x`

**Comment by danielscottt** [comment]
Hi!

I was just looking at issues, and decided to take a look at this one.  Unless I've misunderstood something, I believe this issue to be resolved.  Take the following Richie code:

```
T1 :: BaseType
    x = Integer

T2 :: BaseType
  mem = T1

  :printout
    print mem.x

myt1 = T1
myt1.x = 1

myt2 = T2
myt2.mem = myt1

myt2 printout
```

Which I believe is similar to the example above, and it produces the expected output:

```sh
$ ./ritchie.sh submem.rit
1
```

Am I off base?

**Comment by JoePelz** [comment]
You're right, I'm sorry you had to find out the hard way that it's not an issue anymore.  It looks like I had fixed it in commit 7edf65b2116d58b569388881c92eb856886ad02a 

There was a flurry of activity all at once over the last two weeks before publishing it to the world, and some items got lost in the shuffle

Thanks for letting me know!

**Comment by danielscottt** [comment]
No worries!

Maybe I’ll try another issue!

dan
